src/signature_share.rs

```diff
@@ -0,0 +1,49 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+use reddsa::frost::redjubjub::round2::SignatureShare as FrostSignatureShare;
+
+use crate::{
+    errors::{IronfishError, IronfishErrorKind},
+    participant::{Identity, IDENTITY_LEN},
+};
+
+const FROST_SIGNATURE_SHARE_LEN: usize = 32;
+const SIGNATURE_SHARE_LEN: usize = IDENTITY_LEN + FROST_SIGNATURE_SHARE_LEN;
+
+type SignatureShareSerialization = [u8; SIGNATURE_SHARE_LEN];
+
+pub struct SignatureShare {
+    pub identity: Identity,
+    pub signature_share: FrostSignatureShare,
+}
+
+impl SignatureShare {
```

Comment:
I would add the `Read`/`Write` variants of the serialization methods, both for consistency with the rest of the crate, and to allow better memory usage